components/org.wso2.carbon.identity.scim.common/src/main/java/org/wso2/carbon/identity/scim/common/group/GroupDAO.java

```diff
@@ -86,9 +86,10 @@ public boolean isExistingGroup(String groupName, int tenantId) throws IdentitySC
         boolean isExistingGroup = false;
 
         try {
-            prepStmt = connection.prepareStatement(SQLQueries.CHECK_EXISTING_GROUP_SQL);
+            prepStmt = connection.prepareStatement(SQLQueries.CHECK_EXISTING_ATTRIBUTE_SQL);
             prepStmt.setInt(1, tenantId);
             prepStmt.setString(2, SCIMCommonUtils.getGroupNameWithDomain(groupName));
+            prepStmt.setString(3, SCIMConstants.ID_URI);
```

Comment:
Add comment to indicate, specifically checking SCIM 1.1 ID attribute to avoid conflict with SCIM 2.0